src/server/pfs/server/driver.go

```diff
@@ -1173,8 +1176,9 @@ func (d *driver) listCommit(
 					ci = cis[len(cis)-1-i]
 				}
 				var err error
+				// TODO: Set SizeBytesUpperBound to a sentinel value when the base is not finished.
 				ci.SizeBytesUpperBound, err = d.commitSizeUpperBound(ctx, ci.Commit)
-				if err != nil {
+				if err != nil && !pfsserver.IsBaseCommitNotFinishedErr(err) {
```

Comment:
could use `errors.Is` as it's not crossing a gRPC boundary

Reply:
Yeah, I just went for consistency here. Also, something that I didn't know until recently is that `errors.Is` is a type and value check and `errors.As` is a type check. So, we should use `errors.As` when we only care about the type.